netlify-example/brush/index.html

```diff
@@ -1,388 +1,627 @@
-<!DOCTYPE HTML>
+<!DOCTYPE html>
```

Comment:
Maybe `netlify` should show an `index.html` thanks links to the different kind of examples we're trying to create?

The ability to add to it to quickly test/verify functionality would be really nice. As it stands, you need to know that this URL/page exists.

Reply:
We need to discuss what we are doing in general with the examples, we have multiple types of half finished examples, `jekyll`, etc, lets chat later.